xml/vt_guestfs.xml

```diff
@@ -429,6 +429,38 @@ virt-ls -a sles.qcow2 /tmp/
 .X11-unix
 data.tar</screen>
    </sect3>
+   <sect3 xml:id="sec.guestfs.tools.others.log">
+    <title><command>virt-log</command></title>
+    <para>
+     <command>virt-log</command> shows the log files of the virtual machine
+     disk image or named libvirt domain. If the package
```

Comment:
There are some colons missing, should be "virtual machine,  disk image,  or named libvirt domain."

Is there more relevant info before we move on to Windows, e.g. what package to install to get virt-log? If so, please add it here and make Windows a paragraph of it's own.